Foryoungkids

Monty opened the pull request:
I wanted to use Wordled with grandkids. I needed a shorter list of easier words that would be in their vocabulary. I started by using two lists, wordlist and answers.

I also changed the block font to Arial since they had troubles reading the Fantasy font.

There is one problem I haven't solved. If I change "let answers = " in words.js to an array of strings, chosenWord is now taken from the shorter list, but coloring of blocks no longer works correctly. I'm baffled by this as it doesn't seem the use of chosenWord  in submitWord should need to change. Any ideas?

Monty committed:
Use font more readable for young kids

Monty committed:
Split answer list from word list

Enable creation of shorter answer list

ozboware commented:
Hi @Monty 

I tried to recreate the problem by changing let answers to ```let answers = ['ZORIS', 'WEAVE', 'WONKS'];``` then played the game. It seemed to work okay. Have you got an example of the code you used?

![wordled](https://user-images.githubusercontent.com/95859352/151716546-aeb0467a-f5f3-4469-9ddc-affdc7b56021.png)

Monty committed:
Add answers in second script

Preferred since answers is a separate file

Monty commented:
Thanks for getting back to me. To make it easier I submitted two more pull requests.

In broken1file, I changed “let answers = wordlist” to “let answers = “ a real array of strings.




In broken2files, I added answers.js to index.html and added the same array of strings to the file “answers.js”

Maybe it's some stupid mistake with answers.js. I’ll examine it more closely.




> On Jan 30, 2022, at 12:31 PM, ozboware ***@***.***> wrote:
> 
> 
> Hi @Monty <https://github.com/Monty>
> I tried to recreate the problem by changing let answers to let answers = ['ZORIS', 'WEAVE', 'WONKS']; then played the game. It seemed to work okay. Have you got an example of the code you used?
> 
>  <https://user-images.githubusercontent.com/95859352/151716546-aeb0467a-f5f3-4469-9ddc-affdc7b56021.png>
> —
> Reply to this email directly, view it on GitHub <https://github.com/ozboware/wordled/pull/1#issuecomment-1025224621>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAADFI36IGP2H3OZUYA65TDUYWN3JANCNFSM5NET4Q4Q>.
> Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
> You are receiving this because you were mentioned.
>

Monty committed:
Fix bug with colorizing blocks.

Monty committed:
Tell users how to modify answer list

Monty committed:
Make answers easier to read/edit

Monty commented:
If you’re new to GitHub I strongly recommend reading 
https://docs.github.com/en/get-started/quickstart/github-flow

You can download GitHub Desktop to have the easiest way to follow the process.
https://desktop.github.com

I decided I’d describe how to add your own list of correct words, including  how to use the entire wordlist.

I’ll send a pull request of my topic branch (multiple_lists) for you to check out what I’ve done and merge if you’d like.

If I develop enough Javascript skill, maybe I can figure a way to allow selecting your preferred word list dynamically with either a radio button or menu pick.

Cheers,
Monty

> On Jan 30, 2022, at 4:15 PM, ozboware ***@***.***> wrote:
> 
> 
> Merged #1 <https://github.com/ozboware/wordled/pull/1> into main.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/ozboware/wordled/pull/1#event-5977311516>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAADFI4UEUIR2H4PCVLV2ELUYXICZANCNFSM5NET4Q4Q>.
> Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
> You are receiving this because you were mentioned.
>
